File: typescript/8kyu/Find_Multiples_of_a_Number.ts
export const findMultiples = (integer: number, limit: number): number[] => {
  let totalInteger = integer;
  const result: number[] = [integer];
  while (totalInteger < limit) {
    totalInteger += integer;
    if (totalInteger <= limit) result.push(totalInteger);
  }
  return result;
};

File: rust/src/8kyu/Find_Multiples_of_a_Number.rs
fn find_multiples(n: u32, limit: u32) -> Vec<u32> {
    
    let mut v = Vec::new();
    let mut counter = 1;
    
    while counter*n <= limit{
        v.push(counter*n);
        counter+=1;
    } 
    return v;
}